Userland/Libraries/LibJS/Runtime/ValueInvoke.h

```diff
@@ -0,0 +1,20 @@
+#include <LibJS/Heap/MarkedVector.h>
```

Comment:
Needs license and pragma once

Reply:
a shoot

Reply:
Any reason not to just put this in ValueInlines.h?

Reply:
1: I added a generic license header, as tracking ownership of that function felt a bit tedious
2: This header is a bit more heave than the other inlines, so I feel like keeping it separate is nicer

Reply:
> This header is a bit more heave than the other inlines, so I feel like keeping it separate is nicer

Eh, disagreed, it's now one more special header we need to remember. Seems nicer to me to just have this in ValueInlines.